packages/trader/src/Assets/Trading/Categories/Description/turbos-trade-description.tsx

```diff
@@ -5,7 +5,7 @@ import { Text } from '@deriv/components';
 const TurbosTradeDescription = ({
     onClick,
 }: {
-    onClick: (e?: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>) => void;
+    onClick: (term: string, e?: React.MouseEvent<HTMLElement> | React.KeyboardEvent<HTMLElement>) => void;
```

Comment:
I am fine with type of term being string, but check if this can be made enum, if not  you can ignore it